### ISSUE TITLE
[Docker] Mount volume in directory and not individual file

### ISSUE THREAD

**Issue by dataprolet** [filebrowser/filebrowser#3552]
**Is your feature request related to a problem? Please describe.**
When starting a Docker container where `database.db `and `.filebrowser.json` are mounted as files, the container creates a directory called _database.db_, but not the actual file.

**Describe the solution you'd like**
Mount a volume in a directory, which contains the files instead of the files directly.

**Additional context**
Similar to #3142 which was marked as stale.
Also #3507 is related.

**Comment by charlesangus** [comment]
Agree. Bind-mounting individual files is fiddly and error-prone and imo not a great practice. Just stick both files in a folder called `/config` or something and access them from there.

**Comment by charlesangus** [comment]
No need to mark this as stale, issue continues to exist.

**Comment by Isaak-Malers** [comment]
I also have this issue.  Maybe to preserve backwards compatibility we could have an environment variable for "ConfigDirectory" which defaults to the root of the container.  Then people can specify whatever they want there to match whatever docker conventions they have.

**Comment by yespick0402** [comment]
I think that when docker restarts, the values ​​I set or the account information are initialized, and probably because the database.db and .filebrowser.json I talked about in the topic are not created. Is that right?

If my story is right, can you tell me how to save those files in my directory like the topic of this content and load those files from my directory when docker runs?

**Comment by dataprolet** [comment]
Push

**Comment by laurisvr** [comment]
I managed to get my database in a volume. Here is my docker compose entry:

  filebrowser:
      image: filebrowser/filebrowser:latest
      container_name: filebrowser
      volumes:          # Adjust the host path to the folder you want to serve
        - /data/documents:/srv/documents
        - filebrowser_db:/database
        - ./filebrowser/filebrowser.json:/.filebrowser.json
      ports:
        - "80:80"
      restart: unless-stopped

Then in the same dir as the docker compose i made a dir called filebrowser with 1 file called: filebrowser.json with this content: 
{
    "port": 80,
    "baseURL": "",
    "address": "",
    "log": "stdout",
    "database": "/database/database.db",
    "root": "/srv"
  }

Of course this doesn't allow to you to put the .filebrowser.json in a folder. But to me that's  not as important as the db since it's static.

**Comment by dataprolet** [comment]
> I managed to get my database in a volume.

Thanks, but that's not the issue, because the database is also in a directory inside the container. That still leaves us with the JSON file in the root directory of the container, which has to be mapped as a file.

**Comment by dataprolet** [comment]
Please reopen the issue! Automatically closing issues just because the developers are not responding at all is not how you manage your issues.
